test/integration/backward_compatible/sql/ExecuteTest.js

```diff
@@ -68,7 +68,12 @@ describe('SqlExecuteTest', function () {
     let serverVersionNewerThanFive;
 
     const runSQLQueryWithParams = async () => {
-        for (const _mapName of [mapName, 'public.' + mapName]) {
+        const mapNames = [mapName, `partitioned.${mapName}`];
```

Comment:
Is there a `partitioned` schema in 5.0 anymore?

Reply:
nope, fixed.